axelor-project/src/main/resources/views/ProjectTemplate.xml

```diff
@@ -0,0 +1,109 @@
+			<field name="description" colSpan="12" widget="html" />
+		</panel>
+
+		<panel-tabs name="mainPanelTab">
+			<panel title="Task template" name="taskPanel" showIf="isProject">
+				<field name="taskTemplateSet" widget="TagSelect" colSpan="12"/>
+			</panel>
+
+			<panel title="Members" name="membersPanel">
+				<field name="team" title="Team" readonlyIf="synchronize" colSpan="6" canNew="false" canEdit="false" />
+				<field name="synchronize" colSpan="2" widget="boolean-switch" />
+				<field name="membersUserSet" colSpan="12" widget="TagSelect" canNew="false" form-view="user-form" grid-view="user-grid" hideIf="synchronize" />
+				<button name="importMemberBtn" title="Import all members" showIf="team != null" colSpan="3" onClick="action-project-method-import-members" />
+			</panel>
+
+			<panel title="Invoicing" name="invoicingPanel" showIf="isBusinessProject" if="__config__.app.isApp('business-project')" if-module="axelor-business-project">
+				<field name="teamTaskInvoicing" colSpan="4" onChange="action-project-record-set-invoicing-type"/>
+				<field name="isInvoicingExpenses" colSpan="4" />
+				<field name="isInvoicingPurchases" colSpan="4" />
+				<field name="invoicingType" showIf="teamTaskInvoicing" colSpan="6"/>
+				<field name="invoicingComment" colSpan="12" />
+			</panel>
+
+			<panel title="Wiki" name="wikiPanel" >
+				<panel-related name="wikiListPanel" field="wikiList" colSpan="12" form-view="wiki-form" grid-view="wiki-grid-incl-project"/>
+			</panel>
+
+			<panel title="Configurations" name="configurationsPanel" >
+				<field if="__config__.app.isApp('employee')" if-module="axelor-human-resource" name="productSet" colSpan="12" widget="TagSelect" canEdit="false" domain="self.isActivity = true" form-view="product-activity-form" grid-view="product-activity-grid" />
+				<field name="projectCategorySet" colSpan="12" widget="TagSelect" />
+				<field name="projectFolderSet" colSpan="12" widget="TagSelect" />
+				<field name="imputable" colSpan="4" widget="boolean-switch" if="__config__.app.isApp('employee')" if-module="axelor-human-resource" />
+				<field name="excludePlanning" if-module="axelor-business-project" showIf="isProject" />
+			</panel>
+		</panel-tabs>
+
+		<panel name="actionPanel" sidebar="true">
+			<button name="createProjectBtn" icon="fa-copy" title="Create project from this template" onClick="save,action-project-method-generate-project-view" hidden="true" showIf="!isBusinessProject"/>
+			<button name="createBusinessProjectBtn" icon="fa-copy" title="Create business project from this template" onClick="save,action-project-method-generate-project-view" hidden="true" showIf="isBusinessProject"/>
+		</panel>
+
+		<panel name="characteristicsPanel" title="Characteristics" colSpan="12" sidebar="true" canCollapse="true">
+			<panel name="configPanel" colSpan="12">
+				<field if="__config__.app.isApp('business-project')" if-module="axelor-business-project" name="isProject" colSpan="6" widget="boolean-switch" />
```

Comment:
No need of sub panel put fields directly inside characteristicsPanel , also move all if and if-module condition to that panel.